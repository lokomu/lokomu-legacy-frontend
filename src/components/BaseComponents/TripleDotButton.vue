<template>
  <button
    class="w-8 h-8 bg-white hover:bg-gray-50 border border-blue-100 rounded-md"
    type="button"
  >
    <MoreIcon class="h-7 m-auto rotate-90" />
  </button>
</template>

<script setup lang="ts">
import MoreIcon from "@/components/IconComponents/MoreIcon.vue";
</script>
