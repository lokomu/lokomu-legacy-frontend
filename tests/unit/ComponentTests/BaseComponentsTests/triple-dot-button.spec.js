import { mount } from "@vue/test-utils";
import TripleDotButton from "@/components/BaseComponents/TripleDotButton.vue";
import { describe, expect, it, beforeEach } from "vitest";

describe("TripleDotButton component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(TripleDotButton);
  });

  it("renders correctly", () => {
    expect(wrapper.element).toMatchSnapshot();
  });

  it("is instantiated", () => {
    expect(wrapper.exists()).toBeTruthy();
  });
});
