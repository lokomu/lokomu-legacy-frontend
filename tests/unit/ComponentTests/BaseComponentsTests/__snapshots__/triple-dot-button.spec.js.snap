// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TripleDotButton component > renders correctly 1`] = `
<button
  class="w-8 h-8 bg-white hover:bg-gray-50 border border-blue-100 rounded-md"
  type="button"
>
  <svg
    class="h-7 m-auto rotate-90"
    viewBox="0 96 960 960"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M233.4 620.3q-18.3 0-31.2-13-13-13-13-31.4 0-18.3 13-31.2 13.1-13 31.4-13 18.3 0 31.4 13 13.1 13 13.1 31.4 0 18.3-13.1 31.2-13.1 13-31.6 13Zm246.5 0q-18.3 0-31.2-13-13-13-13-31.4 0-18.3 13-31.2 13-13 31.4-13 18.3 0 31.2 13 13 13 13 31.4 0 18.3-13 31.2-13 13-31.4 13Zm246.5 0q-18.3 0-31.4-13-13.1-13-13.1-31.4 0-18.3 13.1-31.2 13.1-13 31.6-13 18.3 0 31.2 13 13 13 13 31.4 0 18.3-13 31.2-13.1 13-31.4 13Z"
    />
  </svg>
</button>
`;
